tools/modules/software/module_prometheus.sh

```diff
@@ -0,0 +1,96 @@
+module_options+=(
+	["module_prometheus,author"]="@armbian"
+	["module_prometheus,maintainer"]="@efectn"
+	["module_prometheus,feature"]="module_prometheus"
+	["module_prometheus,example"]="install remove purge status help"
+	["module_prometheus,desc"]="Install prometheus container"
+	["module_prometheus,status"]="Active"
+	["module_prometheus,doc_link"]="https://prometheus.io/docs/"
+	["module_prometheus,group"]="Monitoring"
+	["module_prometheus,port"]="9090"
+	["module_prometheus,arch"]="x86-64 arm64"
+)
+#
+# Module prometheus
+#
+function module_prometheus () {
+	local title="prometheus"
+	local condition=$(which "$title" 2>/dev/null)
+
+	if pkg_installed docker-ce; then
+		local container=$(docker container ls -a | mawk '/prometheus?( |$)/{print $1}')
+		local image=$(docker image ls -a | mawk '/prometheus?( |$)/{print $3}')
+	fi
+
+	local commands
+	IFS=' ' read -r -a commands <<< "${module_options["module_prometheus,example"]}"
+
+	PROMETHEUS_BASE="${SOFTWARE_FOLDER}/prometheus"
+
+	case "$1" in
+		"${commands[0]}")
+			pkg_installed docker-ce || module_docker install
+			[[ -d "$PROMETHEUS_BASE" ]] || mkdir -p "$PROMETHEUS_BASE" || { echo "Couldn't create storage directory: $PROMETHEUS_BASE"; exit 1; }
+			
+			# Create dummy prometheus config file if it is not exist
+			if [ ! -f "$PROMETHEUS_BASE/prometheus.yml" ]; then
+  				cat <<- EOF > "$PROMETHEUS_BASE/prometheus.yml"
```

Comment:
Use this way:
https://github.com/armbian/configng/blob/main/tools/modules/software/module_haos.sh#L77
or 
https://github.com/armbian/configng/blob/main/tools/modules/system/module_armbian_upgrades.sh#L103-L108